.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: Report an issue with existing supported devices.
+labels: ["bug"]
+body:
+  - type: input
+    id: info-device
+    attributes:
+      label: Affected Device
+      description: What is the name of the device you are using that is not functioning as expected? If more than one devices are experiencing issues list them here separated by a comma.
+      placeholder: e.g. Corsair Hydro H100i Pro XT
+    validations:
+      required: true
```

Comment:
Some bugs aren't specific to any particular device, so I don't think this should be required.